properties-panel-extension/package.json

```diff
@@ -27,15 +27,20 @@
   "license": "MIT",
   "devDependencies": {
     "babel-core": "^6.26.0",
+    "babel-plugin-transform-flow-strip-types": "^6.22.0",
```

Comment:
Is this one really needed? If so, why?

Maybe migrating to babel@7 is a simpler solution.

Reply:
Could you run the example without this plugin?

I'm gettint this error:
<img width="958" alt="Screenshot 2020-05-28 at 13 43 44" src="https://user-images.githubusercontent.com/15003836/83137327-485b7980-a0e9-11ea-9467-99a5e5be861f.png">

Also, @philippfromme might be interested as well.